modules/rust_examples/RsExample/src/eventually_generated.rs

```diff
@@ -16,6 +16,71 @@ use std::collections::HashMap;
 use std::pin::Pin;
 use std::sync::Arc;
 
+/// Struct holding the config for the `foobar` interface.
+#[derive(Debug)]
+pub struct FoobarConfig {
```

Comment:
Can you not somehow use Serde to deserialize the config structs? Because then you could use `#[serde(default=42)]` which would show up in docs and so on. That would be nice.

Reply:
I think this is not necessary - we aren't parsing a json blob here but get from c++ already one value (either default or an adjusted value)